j/jgrapht/jgrapht_ubi_9_3.sh

```diff
@@ -0,0 +1,72 @@
+#!/bin/bash -e
+# ----------------------------------------------------------------------------
+#
+# Package       : jgrapht
+# Version       : jgrapht-1.5.2
+# Source repo   : https://github.com/jgrapht/jgrapht
+# Tested on     : UBI: 9.3
+# Language      : java
+# Travis-Check  : True
+# Script License: Apache License, Version 2 or later
+# Maintainer    : Stuti Wali <Stuti.Wali@ibm.com>
+#
+#
+# Disclaimer: This script has been tested in root mode on given
+# ==========  platform using the mentioned version of the package.
+#             It may not work as expected with newer versions of the
+#             package and/or distribution. In such case, please
+#             contact "Maintainer" of this script.
+#
+# ----------------------------------------------------------------------------
+
+set -e
+export PACKAGE_NAME=jgrapht
+export PACKAGE_URL=https://github.com/jgrapht/jgrapht
+export LC_ALL=C.UTF-8
```

Comment:
Can you add comment to provide reason for adding these extra steps so for non travis infra if these are not needed user can comment these lines

Reply:
Added comment in script